### ISSUE TITLE
How to use MergeMeshes Node?

### ISSUE THREAD

**Issue by kanomarin** [Luis24989/SkeletalMeshMerger#2]
Can you make a tutorial plz? :)

**Comment by Luis24989** [comment]
Sure I will upload some photos on how it works and get physics mostly working :)

**Comment by kanomarin** [comment]
> Sure I will upload some photos on how it works and get physics mostly working :)

Hi Luis, I  successfully merged several meshes , but I found their collision disappeared.why?

**Comment by Luis24989** [comment]
Yeah I noticed that, as far as I found out it is a problem with the code epic provided for the mesh merge. I will make a tutorial later today where I will show how to enable collision again. But that only works with one physics asset that then gets applied to the skeletal mesh. 
Once I hear anything from epic or other people on the thread I opened in the ue4 forums I will let you know.

Luis
